ydb/core/tx/schemeshard/ut_index_build/ut_vector_index_build.cpp

```diff
@@ -0,0 +1,245 @@
+#include <ydb/core/base/table_index.h>
+#include <ydb/core/tx/schemeshard/ut_helpers/helpers.h>
+#include <ydb/core/tx/schemeshard/schemeshard_billing_helpers.h>
+#include <ydb/core/testlib/tablet_helpers.h>
+
+#include <ydb/core/tx/datashard/datashard.h>
+#include <ydb/core/metering/metering.h>
+
+template <>
+inline void Out<Ydb::Table::IndexBuildState_State>(IOutputStream& o, Ydb::Table::IndexBuildState_State info) {
```

Comment:
Usually it placed here
[ydb/core/protos/out/out.cpp](https://github.com/ydb-platform/ydb/blob/main/ydb/core/protos/out/out.cpp)